incubator/java-spring-boot2/image/Dockerfile-stack

```diff
@@ -1,4 +1,5 @@
 FROM registry.access.redhat.com/ubi8/ubi
+LABEL maintainer="IBM Java Engineering at IBM Cloud"
```

Comment:
We should probably change this or exclude it.

Reply:
Yeah remove I think.